source/designers-guide/custom-detail-page/index.md

```diff
@@ -486,3 +486,17 @@ The code to remove the image zoom is pretty simple. We check if the main `conten
 If you want to take a closer look at all the code we have written in this tutorial you can download this plugin:
 
 <strong><a href="{{ site.url }}/exampleplugins/SwagCustomDetailTheme.zip">Plugin download</a></strong>
+
+The Theme is packed inside a plugin, as that's a requirement to distribute themes via our store.
```

Comment:
The theme is packed inside a plugin. This is a requirement to distribute themes via our store.

Reply:
Done